### PR TITLE
[TASK] Changing trustlines buttons classnames and some css cleanups.

### PR DESCRIPTION
**Pull request by Mesrop**

### CODE REVIEW

File: src/jade/tabs/aud.jade [hunk 1]
```diff
@@ -64,10 +64,10 @@ section.col-xs-12.content(ng-controller='AudCtrl')
 
       .row.fund-tab-description
         .col-xs-7.col-sm-5.col-md-4
-          button.btn.btn-large(ng-show="!audConnected && !loading", type="submit", ng-click="save_account()", ng-disabled="!account.Balance || !can_add_trust") Add Coinex AUD
-          button.btn.btn-large(ng-show="!audConnected && loading", type="submit", ng-disabled="loading") Adding...
-          button.btn.btn-large(ng-show="showInstructions && audConnected", type="submit", ng-click="toggle_instructions()", l10n) Hide instructions
-          button.btn.btn-large(ng-show="audConnected && !showInstructions", type="submit", ng-click="toggle_instructions()", l10n) Show instructions
+          button.btn.btn-large.btn-primary(ng-show="!audConnected && !loading", type="submit", ng-click="save_account()", ng-disabled="!account.Balance || !can_add_trust") Add Coinex AUD
+          button.btn.btn-large.btn-primary(ng-show="!audConnected && loading", type="submit", ng-disabled="loading") Adding...
```
Comment (jublo) [inline src/jade/tabs/aud.jade]: This needs to be localizable.

File: src/jade/tabs/aud.jade [hunk 2]
```diff
@@ -64,10 +64,10 @@ section.col-xs-12.content(ng-controller='AudCtrl')
 
       .row.fund-tab-description
         .col-xs-7.col-sm-5.col-md-4
-          button.btn.btn-large(ng-show="!audConnected && !loading", type="submit", ng-click="save_account()", ng-disabled="!account.Balance || !can_add_trust") Add Coinex AUD
-          button.btn.btn-large(ng-show="!audConnected && loading", type="submit", ng-disabled="loading") Adding...
-          button.btn.btn-large(ng-show="showInstructions && audConnected", type="submit", ng-click="toggle_instructions()", l10n) Hide instructions
-          button.btn.btn-large(ng-show="audConnected && !showInstructions", type="submit", ng-click="toggle_instructions()", l10n) Show instructions
+          button.btn.btn-large.btn-primary(ng-show="!audConnected && !loading", type="submit", ng-click="save_account()", ng-disabled="!account.Balance || !can_add_trust") Add Coinex AUD
```
Comment (jublo) [inline src/jade/tabs/aud.jade]: Is this localizable?

File: src/jade/tabs/btc.jade [hunk 3]
```diff
@@ -78,7 +78,7 @@ section.col-xs-12.content(ng-controller='BtcCtrl')
                       span.modal-error(ng-show="emailError", l10n)
                         | SnapSwap's btc2ripple service is currently unavailable.
                         |  Please check back later.
-                button.btn.btn-large(ng-show="!btcConnected && loading", ng-disabled="loading") Adding...
+                button.btn.btn-primary(ng-show="!btcConnected && loading", ng-disabled="loading") Adding...
```
Comment (jublo) [inline src/jade/tabs/btc.jade]: l10n ftw.

File: src/jade/tabs/eur.jade [hunk 4]
```diff
@@ -64,10 +64,10 @@ section.col-xs-12.content(ng-controller='EurCtrl')
 
       .row.fund-tab-description
         .col-xs-7.col-sm-5.col-md-4
-          button.btn.btn-large(ng-show="!eurConnected && !loading", type="submit", ng-click="save_account()", ng-disabled="!account.Balance || !can_add_trust") Add SnapSwap.eu
-          button.btn.btn-large(ng-show="!eurConnected && loading", type="submit", ng-disabled="loading") Adding...
-          button.btn.btn-large(ng-show="showInstructions && eurConnected", type="submit", ng-click="toggle_instructions()", l10n) Hide instructions
-          button.btn.btn-large(ng-show="eurConnected && !showInstructions", type="submit", ng-click="toggle_instructions()", l10n) Show instructions
+          button.btn.btn-large.btn-primary(ng-show="!eurConnected && !loading", type="submit", ng-click="save_account()", ng-disabled="!account.Balance || !can_add_trust") Add SnapSwap.eu
+          button.btn.btn-large.btn-primary(ng-show="!eurConnected && loading", type="submit", ng-disabled="loading") Adding...
```
Comment (jublo) [inline src/jade/tabs/eur.jade]: Do you speak Esperanto?

File: src/jade/tabs/eur.jade [hunk 5]
```diff
@@ -64,10 +64,10 @@ section.col-xs-12.content(ng-controller='EurCtrl')
 
       .row.fund-tab-description
         .col-xs-7.col-sm-5.col-md-4
-          button.btn.btn-large(ng-show="!eurConnected && !loading", type="submit", ng-click="save_account()", ng-disabled="!account.Balance || !can_add_trust") Add SnapSwap.eu
-          button.btn.btn-large(ng-show="!eurConnected && loading", type="submit", ng-disabled="loading") Adding...
-          button.btn.btn-large(ng-show="showInstructions && eurConnected", type="submit", ng-click="toggle_instructions()", l10n) Hide instructions
-          button.btn.btn-large(ng-show="eurConnected && !showInstructions", type="submit", ng-click="toggle_instructions()", l10n) Show instructions
+          button.btn.btn-large.btn-primary(ng-show="!eurConnected && !loading", type="submit", ng-click="save_account()", ng-disabled="!account.Balance || !can_add_trust") Add SnapSwap.eu
```
Comment (jublo) [inline src/jade/tabs/eur.jade]: I love Spanish.

File: src/jade/tabs/gold.jade [hunk 6]
```diff
@@ -62,10 +62,10 @@ section.col-xs-12.content(ng-controller='GoldCtrl')
 
       .row.fund-tab-description
         .col-xs-7.col-sm-5.col-md-4
-          button.btn.btn-large(ng-show="!gbiConnected && !loading", type="submit", ng-click="save_account()", ng-disabled="!account.Balance || !can_add_trust") Add GBI
-          button.btn.btn-large(ng-show="!gbiConnected && loading", type="submit", ng-disabled="loading") Adding...
-          button.btn.btn-large(ng-show="showInstructions && gbiConnected", type="submit", ng-click="toggle_instructions()") Hide instructions
-          button.btn.btn-large(ng-show="gbiConnected && !showInstructions", type="submit", ng-click="toggle_instructions()") Show instructions
+          button.btn.btn-large.btn-primary(ng-show="!gbiConnected && !loading", type="submit", ng-click="save_account()", ng-disabled="!account.Balance || !can_add_trust") Add GBI
```
Comment (jublo) [inline src/jade/tabs/gold.jade]: Well, I will shut up now.

File: src/jade/tabs/gold.jade [hunk 7]
```diff
@@ -62,10 +62,10 @@ section.col-xs-12.content(ng-controller='GoldCtrl')
...
+          button.btn.btn-large.btn-primary(ng-show="!gbiConnected && loading", type="submit", ng-disabled="loading") Adding...
```
Comment (jublo) [inline src/jade/tabs/gold.jade]: You know where to find all those.

**Comment by Mesrop** [comment]
@jublo :D :D thanks, done! ;)

**Comment by vhpoet** [comment]
LGTM

**Comment by mrajvanshy** [comment]
LGTM

**Pull request merged**